src/misc/simulation/fscommon/fscommonutil.cpp

```diff
@@ -283,7 +283,7 @@ namespace swift::misc::simulation::fscommon
             if (!ok)
             {
                 msgs.push_back(CStatusMessage(static_cast<CFsCommonUtil *>(nullptr))
-                                   .validationWarning(u"Model '%1' '%2' in none of the %3 SimObjects path(s)")
+                                   .validationWarning(u"Model '%1' '%2' in none of the %3 SimObjects path(s) %3 ")
```

Comment:
Why adding the additional ``%3`` at the end?

Reply:
to display the paths against which the model set is checked. These sometimes do not match the set paths. Can be removed at a later date.

Reply:
But isn't %3 already printed in the string? Now its printed twice.